examples/all-clusters-app/ameba/main/include/static-supported-modes-manager.h

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
If all the instances of all-clusters-app use the same exact code, could we have just one copy of it in `examples/all-clusters-app/all-clusters-common/src` instead of one copy per platform?  I can understand not wanting to put this into `src/app/clusters`, because this is very app-specific code, but it's not specific to the (app, platform) pair...

I see @jmartinez-silabs asked this question too...  the answer is that people didn't do the work to do things right, preferrring to copy/paste instead.  Let's not keep doing that, please, if it can be avoided.